src/server.py

```diff
@@ -34,6 +34,22 @@ def get_covid_data_by_state(state):
                             mimetype='application/json')
     return response
 
+@app.route('/groundhog/v1/counties', methods=['GET'])
+def get_county_map():
+    county_json = CSVService.get_state_county_info()
+    response = Response(response=json.dumps(county_json),
```

Comment:
let's add error handling here for NOT FOUND errors.

Reply:
Does it make sense to add NOT FOUND errors and return a 404 since there is no input parameters, and therefore no way for it to not find what it needs?

Reply:
you are correct, i misread that